hubtools/src/caboose.rs

```diff
@@ -130,6 +152,7 @@ impl CabooseBuilder {
             (tags::NAME, self.name),
             (tags::VERS, self.version),
             (tags::SIGN, self.sign),
+            (tags::EPOC, self.epoch),
```

Comment:
...then call `to_string()` on the `epoch: u32` here.

Reply:
yup